### PR TITLE
Malware archive 

### PR DESCRIPTION
**Pull request by cccs-sgaron**

### CODE REVIEW

File: src/components/visual/ArchiveDetail/viewer/ascii.tsx
Comment (cccs-sgaron) [inline src/components/visual/ArchiveDetail/viewer/ascii.tsx:1]: We should not duplicate en should reuse the file viewer component

File: src/components/visual/ArchiveDetail/viewer/hex.tsx
Comment (cccs-sgaron) [inline src/components/visual/ArchiveDetail/viewer/hex.tsx:1]: We should not duplicate en should reuse the file viewer component

File: src/components/visual/ArchiveDetail/viewer/strings.tsx
Comment (cccs-sgaron) [inline src/components/visual/ArchiveDetail/viewer/strings.tsx:1]: We should not duplicate en should reuse the file viewer component

File: src/locales/en/file/detail.json
Comment (cccs-sgaron) [inline src/locales/en/file/detail.json:1]: Do not add archive related translation strings to file details

File: src/locales/fr/file/detail.json
Comment (cccs-sgaron) [inline src/locales/fr/file/detail.json:1]: Do not add archive related translation strings to file details

**Review by cccs-nr — APPROVED** [review]

**Pull request merged**